astropy/table/operations.py

```diff
@@ -697,6 +697,69 @@ def common_dtype(cols):
         raise tme
 
 
+def _get_join_sort_idxs(out_descrs, keys, len_left, len_right, left, right):
```

Comment:
Looking again I think that `out_descrs` isn't needed, and `len_left` and `len_right` could be dropped.

Reply:
Agreed.